_opensearch/supported-field-types/completion.md

```diff
@@ -0,0 +1,380 @@
+---
+layout: default
+title: Completion
+nav_order: 51
+has_children: false
+parent: Autocomplete field types
+grand_parent: Supported field types
+---
+
+# Completion field type
+
+A completion field type is a completion suggester that provides auto-complete functionality. It is a prefix suggester, so it matches the beginning of text only. Completion suggester creates an in-memory data structure, which provides faster lookups but leads to increased memory usage. You need to upload a list of all possible completions into the index before using this feature.
```

Comment:
disregard if "auto-complete functionality" is proper terminology.
```suggestion
A completion field type is a completion suggester that provides auto-complete functionality. It is a prefix suggester, so it matches the beginning of text only. Completion suggester creates an in-memory data structure, which provides faster lookups but leads to increased memory usage. You need to upload a list of all possible completions into the index before using this feature.
```

Reply:
Changed